src/comm/tests/broadcast.rs

```diff
@@ -22,7 +22,7 @@ use ore::future::OreTryStreamExt;
 
 /// Verifies that broadcast tokens can allocate channels dynamically by
 /// overriding the `Token::uuid` method.
-#[tokio::test]
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
why did this change? asking because this smells like a functional change rather than a mechanical refactor but maybe thats totally wrong

Reply:
Ah, yeah, something in Tokio that gets used in the `comm` package now requires the multi-threaded runtime. Our usage of `comm` in production always uses the multi-threaded runtime, so didn't seem too worth tracking down.